jobs/pending_payments.ts

```diff
@@ -38,6 +39,7 @@ exports.attemptPendingPayments = async (bot: Telegraf<MainContext>): Promise<voi
         request: pending.payment_request,
       });
       const buyerUser = await User.findOne({ _id: order.buyer_id });
+      if (buyerUser === null) return;
```

Comment:
@Mersho why are you changing behaviour here? converting from JS to TS should not change the behaviour of the program